### PR TITLE
Default Router macros `plug_opts` to empty list

### PR DESCRIPTION
**Pull request by scrogson**

### CODE REVIEW

File: test/phoenix/router/routing_test.exs [hunk 1]
```diff
@@ -34,6 +39,8 @@ defmodule Phoenix.Router.RoutingTest do
     match :move, "/move", UserController, :move
 
     get "/users/:user_id/files/:id", UserController, :image
+    get "/standard", StandardPlug
```
Comment (chrismccord) [inline test/phoenix/router/routing_test.exs]: let's use a function plug here instead so we don't have to create another module

**Comment by josevalim** [comment]
I am :-1:. The most common usage is to include an action. With this change, folks will write `get "/foo", PostController` and it won't work. I think being explicit here is worth it. :)

**Comment by scrogson** [comment]
@josevalim explicit as in `get "/foo", MyPlug, []` ???

**Comment by chrismccord** [comment]
Yes, exactly

**Comment by josevalim** [comment]
Yes, because in the majority of cases not passing the options will lead to the wrong result.

**Comment by scrogson** [comment]
Cool...let's close this then.

**Pull request closed without merging**